comp/logs/agent/config/integration_config.go

```diff
@@ -35,59 +35,59 @@ const (
 // LogsConfig represents a log source config, which can be for instance
 // a file to tail or a port to listen to.
 type LogsConfig struct {
```

Comment:
A lot of these settings are set internally and not valid to be set by the user. If we are surfacing this config to the user, do we really want to include all of the internal properties? 

such as (no exhaustive list)
- `Type`
- `Identifier`
- `Query`
- `Image`
- `Label`

I just want to make sure we are not in a position where the user sees all of these config keys and thinks they can change them in the agent.

Reply:
See https://github.com/DataDog/datadog-agent/pull/21563#discussion_r1430162495